packages/ontario-design-system-component-library/src/components/ontario-task/ontario-task.tsx

```diff
@@ -0,0 +1,221 @@
+import { h, Component, Prop, Watch, State, Listen } from '@stencil/core';
+import { ConsoleMessageClass } from '../../utils/console-message/console-message';
+import { TaskStatuses, TaskStatus, BadgeColors } from './ontario-task-statuses';
+import { validateLanguage, validateValueAgainstArray } from '../../utils/validation/validation-functions';
+import { Hint } from '../../utils/common/common.interface';
+import { Language } from '../../utils/common/language-types';
+import { constructHintTextObject } from '../../utils/components/hints/hints';
+import translations from '../../translations/global.i18n.json';
+
+@Component({
+	tag: 'ontario-task',
+	styleUrl: 'ontario-task.scss',
+	shadow: true,
+})
+export class OntarioTask {
+	hintTextRef: HTMLOntarioHintTextElement | undefined;
+
+	/**
+	 * Specifies the label of the task.
+	 *
+	 * This is required to provide the name of the task.
+	 */
+	@Prop() label: string;
+
+	/**
+	 * Specifies an optional link associated with the task.
+	 *
+	 * If provided, clicking the task will navigate to this URL.
+	 */
+	@Prop() link?: string;
```

Comment:
I wonder if we should call this `href` rather than `link`, thoughts?

Reply:
I prefer link 🤔

Reply:
I agree with Scott re: href.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#href

If this prop is just the url, then href is most appropriate. This is the "linked URL".